src/core/include/openvino/core/shape.hpp

```diff
@@ -37,6 +39,7 @@ class Shape : public std::vector<size_t> {
 
     OPENVINO_API Shape& operator=(const Shape& v);
     OPENVINO_API Shape& operator=(Shape&& v) noexcept;
+    std::string to_string() const;
```

Comment:
```suggestion
    OPENVINO_API std::string to_string() const;
```
There's a linker error when I call `to_string()` from benchmark_app without it.

Reply:
Applied, thanks.